dl/src/main/scala/com/intel/analytics/bigdl/optim/DistriOptimizer.scala

```diff
@@ -238,10 +239,12 @@ object DistriOptimizer {
         accumulateCount += recordsNum.value
         val end = System.nanoTime()
         wallClockTime += end - start
+        state.getOrElse("learningRateSchedule", Default())
+          .updateHyperParameter(state, driverState)
         logger.info(s"${_header} Train ${recordsNum.value} in ${(end - start) / 1e9}seconds. " +
           s"Throughput is ${recordsNum.value / ((end - start) / 1e9)} records/second. Loss is ${
             lossSum.value / finishedModelNum
-          }. ")
+          }. Current learning rate is ${-state[Double]("clr")}.")
```

Comment:
Consider move this into a new method of OptimMethod. I think it should not be exposed here.